### PR TITLE
Update CrowdStrikeFalcon.yml

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@JasBeilin 
Following are several questions for the yml file:

- Line 92 - What integration doc?  We need a link here.
- Line 99 -  What integration doc?  We need a link here.
- line 303 - Where do you get the behavior ID from?
- Line 2697 - In what format is the date?
- Line 2735 - I don't think that isArray should be true.
- Line 3178 - I could not access the link https://www.falconpy.io/Service-Collections/Falcon-Query-Language
- Line 3682 - No description
- Line 4956 - What is TTO?

For the following lines, isArray: true, but the description is not A comma-separated list of.....
Should it be?
line 346
Line 444
Line 779
Line 786
Line 820
Line 977
Line 1456
Line 1459
Line 2433
Line 2611
Line 2614
Line 2707
Line 2710
Line 2713
Line 2716
Line 2719
Line 2724
Line 2727
Line 2729
Line 2913
Line 2936
Line 4040
Line 4043
Line 4046
Line 4049
Line 4052
Line 4207
Line 4211
Line 4214
Line 4217
Line 4384
Line 4387
Line 4694
Line 4712
Line 4749
Line 4786
Line 4822
Line 4959

**Comment by BEAdi** [comment]
* _Line 92 - What integration doc?  We need a link here._
This one: https://falcon.crowdstrike.com/documentation/page/d3c84a1b/falcon-query-language-fql

* _Line 99 -  What integration doc?  We need a link here._
This one: https://falcon.crowdstrike.com/documentation/page/d3c84a1b/falcon-query-language-fql

* _line 303 - Where do you get the behavior ID from?_
From the command `cs-falcon-search-detection` or `cs-falcon-get-detections-for-incident`.

* _Line 2697 - In what format is the date?_
It is the UTC timestamp in seconds.

* _Line 2735 - I don't think that isArray should be true._
I agree.

* _Line 3178 - I could not access the link https://www.falconpy.io/Service-Collections/Falcon-Query-Language_
We can use this one: https://falcon.crowdstrike.com/documentation/page/d3c84a1b/falcon-query-language-fql here too. It is accessible only after logging in (I opened it).

* _Line 3682 - No description_
It is a hidden argument so it is not viewable from the UI. Anyway, we can add this description: "Whether to hide the polling message and only print the final status at the end (automatically filled by polling, can be used for testing purposes)."

* _Line 4956 - What is TTO?_
Time to Operate.

* _For the following lines, `isArray: true`, but the description is not `A comma-separated list of.....` Should it be?_
  * No: 2433, 2611, 2614.
  * In lines 2913, 2936 I think that no because of the end of the description ("This filter supports multiple values and negation."), but it is up to you as both will be correct. 
  * In all other places- yes.
  * In line 786 `isArray: true` should be added.

**Comment by ShirleyDenkberg** [comment]
@BEAdi I implemented all of your answers to my questions.

**Review by BEAdi — APPROVED** [review]
Looks amazing!

**Review by BEAdi — COMMENTED** [review]
@ShirleyDenkberg I regenerated the REAMDE with the changes made to the yml, can you go over it too? There are some sections and more information in addition to the yml descriptions.

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 1]
```diff
@@ -56,28 +57,26 @@ In order to use the CrowdStrike Falcon integration, your API client must be prov
...
+    | --- | --- |
+    | None | Turns off incident mirroring. |
+    | Incoming | Any changes in CrowdStrike Falcon events (mirroring incoming fields) will be reflected in Cortex XSOAR incidents. |
+    | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in CrowdStrike Falcon events (outgoing mirrored fields). |
+    | Incoming And Outgoing | Changes in Cortex XSOAR incidents and CrowdStrike Falcon events will be reflected in both directions. |
+
+5. Optional: Check the *Close Mirrored XSOAR Incident* integration parameter to close the Cortex XSOAR incident when the corresponding event is closed in CrowdStrike Falcon.
+6. Optional: Check the *Close Mirrored CrowdStrike Falcon Incident or Detection* integration parameter to close the CrowdStrike Falcon incident or detection when the corresponding Cortex XSOAR incident is closed.
+
+Newly fetched XSOAR incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:79]:
```suggestion
Newly fetched Cortex XSOAR incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 2]
```diff
@@ -1035,7 +1039,7 @@ Runs a script on the agent host.
 >| runscript | runscript -Raw=Write-Output 'Hello, World! | a1a1a1a1a1a1a1a1a1a1a1a1a1a1a1a1 |  | Hello, World! |                                    Type         Size (bytes)    Size (MB)       Last Modified (UTC-5)     Created (UTC-5)          <br/>----                                     ----         ------------    ---------       ---------------------     ---------------          <br/>$Recycle.Bin                             &lt;Directory&gt;  --              --              11/27/2018 10:54:44 AM    9/15/2017 3:33:40 AM     <br/>ITAYDI                                   &lt;Directory&gt;  --              --              11/19/2018 1:31:42 PM     11/19/2018 1:31:42 PM     |
 
 
-### 17. cs-falcon-run-get-command
+### cs-falcon-run-get-command
 
 ***
 Batch executes 'get' command across hosts to retrieve files.
```
[the remark this answers is not in the document]
Comment (ShirleyDenkberg) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:1043]: "The running status you requested in the get command can be checked with cs-falcon-status-get-command."

**Comment by ShirleyDenkberg** [comment]
@BEAdi Integration Readme was reviewed.

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 3]
```diff
@@ -6728,94 +6165,339 @@ Get incident behavior information
...
+| CVE ID | Host Info hostname | Host Info os Version | Host Info Product Type Desc | Host Info Local IP | Host Info ou | Host Info Machine Domain | Host Info Site Name | CVE Exploitability Score | CVE Vector |
+| --- | --- | --- | --- |  --- | --- |  --- | --- |  --- | --- |
+| CVE-20212-2222 |  host | 1 | Server | ip |  |  | site | 5.5 |  |
 
-### Troubleshooting
+## Troubleshooting
 When encountering connectivity or authorization errors within Cortex XSOAR 8, 
-it necessary to include the IP corresponding to the relevant region into the CrowdStrike Falcon allow list.
\ No newline at end of file
+it necessary to include the IP corresponding to the relevant region into the CrowdStrike Falcon allow list.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:6503]:
```suggestion
it is necessary to include the IP address corresponding to the relevant region in the CrowdStrike Falcon allow list.
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 4]
```diff
@@ -137,10 +136,14 @@ Available parameters:
 Exmample: `cloud_provider=aws&region=eu-west-2`
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:136]:
```suggestion
Example: `cloud_provider=aws&region=eu-west-2`
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 5]
```diff
@@ -117,7 +116,7 @@ Available filter:
 
 Exmample: `cloud_provider: 'aws'+account_id: 'my_id'`
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:117]:
```suggestion
Example: `cloud_provider: 'aws'+account_id: 'my_id'`
```

**Comment by BEAdi** [comment]
Force merge- the PR only has documentation changes, and therefore is not related to the playbook failure (only on NG, on all other servers it does pass).

**Pull request merged**